.gitignore

```diff
@@ -1,3 +1,4 @@
 *.exe
+.idea
```

Comment:
this should go in your global gitignore, instead of in specific project ones

Reply:
Done